libs/LzmaLib/source/MtCoder.h

```diff
@@ -107,7 +107,7 @@ typedef struct _CMtCoder
     SRes writeRes;
     unsigned writeIndex;
     Byte ReadyBlocks[MTCODER__BLOCKS_MAX];
-    LONG numFinishedThreads;
+    Int32 numFinishedThreads;
```

Comment:
This change might not be a good idea, since this is [LZMA SDK 19.00](https://www.7-zip.org/sdk.html) code - my apologies if my previous comment was confusing.
Might be a better idea to keep the LONG define in 7zTypes.h :)